sanoid

```diff
@@ -181,6 +181,10 @@ sub monitor_snapshots {
 					if (! $config{$section}{'monitor_dont_warn'} && ($errorlevel < 2) ) { $errorlevel = 1; }
 					push @msgs, "WARN: $path newest $type snapshot is $dispelapsed old (should be < $dispwarn)";
 				}
+			} elsif ($elapsed < 0) {
```

Comment:
a potential undefined value is compared here and will probably trigger a warning